src/versions/git_helpers.ts

```diff
@@ -42,7 +42,9 @@ export function getPrevVersion(): string {
  * Returns the "previous" version tag according to the rules described in `run`.
  */
 export function getPrevVersionTag(): string {
-  return `v${getPrevVersion()}`;
+  const previousVersion = getPrevVersion();
+
+  return previousVersion != null ? `v${previousVersion}` : null;
```

Comment:
Als Anregung: Bei Ternary-Operatoren ist es für Neulinge im Projekt immer leichter Gleichungen als Ungleichungen "mental zu parsen".

Vielleicht ein Stück weit Ansichtssache, aber mein Vorschlag wäre:

```suggestion
  return previousVersion == null ? null : `v${previousVersion}`;
```

Reply:
Ok versteh ich. 👍
Für mich ist es leichter, wenn erst der Wert den ich eigentlich erwarte und dann der Wert bei dem etwas schief gelaufen ist zugewiesen wird.
Aber das ist vermutlich von Person zu Person unterschiedlich.